static/app/components/smartSearchBar/index.tsx

```diff
@@ -1514,16 +1540,35 @@ class SmartSearchBar extends Component<DefaultProps & Props, State> {
     }
 
     const cursorSearchTerm = this.cursorSearchTerm;
+
     if (cursorToken.type === Token.FreeText && cursorSearchTerm) {
-      const autocompleteGroups = [
-        await this.generateTagAutocompleteGroup(cursorSearchTerm.searchTerm),
-      ];
+      const groups: AutocompleteGroup[] | null =
+        cursorToken.invalid?.type === InvalidReason.WILDCARD_NOT_ALLOWED
+          ? [
+              {
+                searchItems: [
+                  {
+                    type: ItemType.INVALID_QUERY_WITH_WILDCARD,
+                    desc: cursorSearchTerm.searchTerm,
+                    callback: () =>
+                      window.open(
+                        'https://docs.sentry.io/product/sentry-basics/search/searchable-properties/'
+                      ),
+                  },
+                ],
+                recentSearchItems: [],
+                tagName: cursorSearchTerm.searchTerm,
+                type: ItemType.INVALID_QUERY_WITH_WILDCARD,
+              },
+            ]
+          : [await this.generateTagAutocompleteGroup(cursorSearchTerm.searchTerm)];
```

Comment:
Is there a better way to do this without these special one off conditionals? Like should this actually happen in `generateTagAutocompleteGroup`

Reply:
Yes, it is indeed possible to move it there and it would cover all invalid cases (did an experiment yesterday). However, due to the numerous test failures it caused, I ultimately decided to maintain the current state of the code.